### PR TITLE
update model to new codegen

### PR DESCRIPTION
**Pull request by danrubel**
This updates the model based on the latest codegen.

These changes are generated by the same codegen that now generates the nook model.

The model can be regenerated by loading the `dan-required-field-values` branch of the `Infrastructure` project
and then running `katikati-reporting/tool/build_model.sh`

### CODE REVIEW

**Review by marianamarasoiu — APPROVED** [review]
LGTM from me, but I believe these codegen changes intersect with other substantial changes to the model structure that @elayabharath is working on in this repo (so the classes etc. might look a bit different after those land). @elayabharath - best to leave you to merge this when ready, or might be easiest to re-generate this code when the new model structure is ready?

**Comment by danrubel** [comment]
Obsolete... included as part of https://github.com/larksystems/katikati-reporting/pull/57

**Pull request closed without merging**
